pyfr/solvers/euler/inters.py

```diff
@@ -1,7 +1,12 @@
+from pyfr.mpiutil import mpi
+from pyfr.plugins.base import init_csv
```

Comment:
Awkward dependency (solvers shouldn't depend on plugins which are higher level).

Reply:
I could move `init_csv` (maybe also `open_hdf5_a` and `DatasetAppender`?) to writers.base? Though they are different kinds of 'writers' than what that directory is aimed at

Reply:
I've moved `init_csv`, `open_hdf5_a` and `DatasetAppender` to writers.timeseries. The other option would be to have a mass flow stats plugin that would handle writing out a csv, let me know if that would be better instead.